### ISSUE TITLE
Allow option to only sync finished order

### ISSUE THREAD

**Issue by ankush** [frappe/ecommerce_integrations#57]
Some users don't want to be bothered by SO at the creation level and just want the final invoice/inventory impact in their ERP.

Option to sync completed orders only serves this use case.

**Comment by federicocalvo** [comment]
https://github.com/frappe/bench/pull/1195

That may be?

**Comment by ankush** [comment]
@federicocalvo doesn't seem relevant... wrong repo?

**Comment by federicocalvo** [comment]
Yes sorry 😅, it was for https://github.com/frappe/ecommerce_integrations/issues/58

**Comment by ankush** [comment]
Still not relevant, that issue is solvable with current queues.
